platform/ui/tailwind.config.js

```diff
@@ -166,6 +166,9 @@ module.exports = {
       '48': '12rem',
       '56': '14rem',
       '64': '16rem',
+      '72': '18rem',
+      '80': '20rem',
+      '250px': '250px',
```

Comment:
Only note, make sure these styles aren't specific for `docz` theme and not used in our actual design.